internal/web/app/assets/css/tailwind.css

```diff
@@ -1,2 +1 @@
 @import "tailwindcss";
-@import "flowbite/src/themes/default";
```

Comment:
why do we remove flowbite?

Reply:
because it wasn't used anywhere anymore. It was only for the info popovers which are not implemented with just css and alpinejs, so it's one less depedency (it did remove quite a lot of things from package-lock.json)

Reply:
noice!